webpack.config.js

```diff
@@ -0,0 +1,10 @@
+const path = require('path');
+
+module.exports = {
```

Comment:
- [ ] Add the devserver output location so the `npm run start` command works on all systems.
```
  devServer: {
    publicPath: '/',
    contentBase: './dist',
    hot: true
}
```